projects/nblocks-ionic/src/lib/shared/lib-config.ts

```diff
@@ -32,6 +43,15 @@ export const defaultLibConfig = (config: Partial<LibConfig>): LibConfig => {
     openRoutes: ["/"],
     roles: ["OWNER", "ADMIN", "MANAGER", "VIEWER"],
     languages: ['en','sv'],
+    socialLogins: {
+      accountApiHost: "http://localhost:3010",
```

Comment:
Default value should be `https://account-api.nebulr-core.com` and not something we use to develop locally.